airflow/providers/amazon/aws/operators/s3_bucket.py

```diff
@@ -37,12 +37,16 @@ class S3CreateBucketOperator(BaseOperator):
         running Airflow in a distributed manner and aws_conn_id is None or
         empty, then default boto3 configuration would be used (and must be
         maintained on each worker node).
+        
```

Comment:
Not sure this editorializing should be in here (same with the other files)?

Reply:
Its good idea to give some info as part of the comments so that users are aware why aws_conn_id was added as part of the template_fields and what other benefit it could have.